docs/ape17.rst

```diff
@@ -496,11 +496,10 @@ recommended approach is to use the tox file to set up the different
 configurations you want to use, and to then keep the CI configuration as simple
 as possible.
 
-If you use Travis CI, a good place to start is the ``.travis.yml`` file
-generated in :ref:`Step 0 <step-rerender>`, and you can then see if any previous customizations you had
-made need to be copied over. This file shows how one can configure Travis to use
-tox, optionally using conda via ci-helpers to set up Python on MacOS X and
-Windows.
+Github now has an integrated CI service, Github Actions. If you wish to use Actions, a good place to start
```

Comment:
Officially spelling is "GitHub" with a capital "H" so I think we need to follow that throughout for consistency?
```suggestion
GitHub now has an integrated CI service, GitHub Actions. If you wish to use Actions, a good place to start
```

Reply:
yeah, i see it done both ways, but the official docs use the H so we should, too.